pom.xml

```diff
@@ -9,10 +9,10 @@
 		<relativePath/> <!-- lookup parent from repository -->
 	</parent>
 	<groupId>com.projects</groupId>
-	<artifactId>Employee_management_system</artifactId>
-	<version>0.0.1-SNAPSHOT</version>
+	<artifactId>EmployeeManagementSystem</artifactId>
+	<version>0.1.0-SNAPSHOT</version>
```

Comment:
Do not use SNAPSHOT at the end. Just 0.1.0

Reply:
updated